src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizerFactory.cs

```diff
@@ -157,11 +159,13 @@ public IStringLocalizer Create(string baseName, string location)
 
             location = location ?? _hostingEnvironment.ApplicationName;
 
-            baseName = GetResourcePrefix(baseName, location);
```

Comment:
Not using this `protected virtual` method is a breaking change. Those who have overridden it will no longer see their preferred behaviour.